django/utils/version.py

```diff
@@ -9,8 +9,6 @@
 # or later". So that third-party apps can use these values, each constant
 # should remain as long as the oldest supported Django version supports that
 # Python version.
-PY36 = sys.version_info >= (3, 6)
```

Comment:
Following the comment's instructions, PY36 shouldn't be removed until Django 2.2 (which supports Python 3.5) is no longer supported. PY37 can be removed when Django 3.2 (which supports Python 3.6) is no longer supported.

Reply:
Thanks @timgraham! #13996.